targets/TARGET_NORDIC/TARGET_NRF5/us_ticker.c

```diff
@@ -52,20 +53,27 @@
 bool              m_common_rtc_enabled = false;
 uint32_t volatile m_common_rtc_overflows = 0;
 
-#if defined(TARGET_MCU_NRF51822)
-void common_rtc_irq_handler(void)
-#else
-void COMMON_RTC_IRQ_HANDLER(void)
-#endif
+__STATIC_INLINE uint32_t rtc_ovf_evet_check(void)
 {
-
     if (nrf_rtc_event_pending(COMMON_RTC_INSTANCE, NRF_RTC_EVENT_OVERFLOW)) {
         nrf_rtc_event_clear(COMMON_RTC_INSTANCE, NRF_RTC_EVENT_OVERFLOW);
         // Don't disable this event. It shall occur periodically.
 
         ++m_common_rtc_overflows;
     }
 
+    return m_common_rtc_overflows;
```

Comment:
This is a bit ineffective. This variable is volatile, so it will be re-read every time, and the return value from the first call of this function is not used at all (see line 75). It would be better to not return anything here and read _m_common_rtc_overflows_ when it is really needed (i.e. in line 182).